lib/loginUI.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+
+class LoginPage extends StatefulWidget {
+  static String tag = 'login-page';
+  @override
+  _LoginPageState createState() => new _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    final branding = Hero(
+      tag: 'hero',
+      child: Text(
+        'amFOSS CMS',
+        style: TextStyle(color: Colors.black87, fontWeight: FontWeight.bold, fontSize: 50),
+        textAlign: TextAlign.center
+      ),
+    );
+
+    final username = TextFormField(
+      autofocus: false,
+      decoration: InputDecoration(
+        hintText: 'Username',
+        contentPadding: EdgeInsets.fromLTRB(20.0, 10.0, 20.0, 10.0),
+        border: OutlineInputBorder(borderRadius: BorderRadius.circular(32.0)),
+      ),
+    );
+
+    final password = TextFormField(
+      autofocus: false,
+      obscureText: true,
+      decoration: InputDecoration(
+        hintText: 'Password',
+        contentPadding: EdgeInsets.fromLTRB(20.0, 10.0, 20.0, 10.0),
+        border: OutlineInputBorder(borderRadius: BorderRadius.circular(32.0)),
+      ),
+    );
+
+    final loginButton = Padding(
+      padding: EdgeInsets.symmetric(vertical: 16.0),
+      child: RaisedButton(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(24),
+        ),
+        padding: EdgeInsets.all(12),
+        color: Colors.lightBlueAccent,
+        child: Text('Log In', style: TextStyle(color: Colors.white)),
+      ),
+    );
+
+    final forgotLabel = FlatButton(
```

Comment:
Instead of having a FlatButton, you could use an Inkwell widget, or better yet a GestureDetector. This would make the UX much better.

Reply:
Updated it. Take a look @padth4i

Reply:
Looks good.